openvdb/points/AttributeArray.h

```diff
@@ -256,17 +256,17 @@ class OPENVDB_API AttributeArray
     template<typename ValueType>
     bool hasValueType() const { return this->type().first == typeNameAsString<ValueType>(); }
 
-    /// Set value at given index @a n from @a sourceIndex of another @a sourceArray
-    /// @note deprecated from ABI=6, use copyValues() with source-target index pairs
+    /// @brief Set value at given index @a n from @a sourceIndex of another @a sourceArray.
+    /// @deprecated From ABI 6 on, use copyValues() with source-target index pairs.
```

Comment:
Tag deprecation-related notes with `@deprecated`.

Reply:
Ah, I wasn't aware of this tag, I'll use this in future, thanks.